js-api-doc/README.md

```diff
@@ -1,13 +1,21 @@
-The [`sass` package] on npm is a pure-JavaScript package built from the [Dart
-Sass] implementation. In addition to Dart Sass's [command-line interface], it
-provides a JavaScript API that can be used to drive Sass compilations from
-JavaScript. It even allows an application to control {@link Options.importers |
+The Sass JavaScript API can be used to to drive Sass Compilations from JavaScript. It even allows an application to control {@link Options.importers |
```

Comment:
Nit: Reflow this to match the 80-character line length.

Reply:
Fixed in c8b1198eeb33347f8c5df28e6a5ff798d5375c09